CHANGELOG.md

```diff
@@ -0,0 +1,129 @@
+# [1.0.6] - 2021-03-31
```

Comment:
Just a suggestion: I personally find it really helpful when these version numbers link to the actual tag on Github. It can facilitate hunting regression bugs a lot.

I'm not sure how the release process for this repo works 🤷 But I guess tags / releases are involved somehow, right?

Reply:
Good idea! New issue: #463

Right now it’s all very manual — suggestions on making that smoother & easier are definitely welcome :) I’m considering something [conventional-commits](https://www.conventionalcommits.org/en/v1.0.0/) and automating the changelog generation, wdyt?